Updates in Turkish translation

yamac-kurtulus opened the pull request:
I improved the Turkish translation a little bit, fixed the links in there and added Turkish to the translation list.

yamac-kurtulus committed:
Improved Turkish Translation,
Fixed a few minor typos,
Added the translation to translations.md

yamac-kurtulus committed:
Translations.md Turkish link fixed

yamac-kurtulus committed:
Fixing some of the links

yamac-kurtulus committed:
Fixed links on tutorial

yamac-kurtulus committed:
Last proofread

sara-02 commented:
@yamac-kurtulus I just realised that we have another Turkish translation PR https://github.com/Roshanjossey/first-contributions/pull/3353
@karsibali can you guys work together to improve this translation?

yamac-kurtulus commented:
Thanks for reminding me @sara-02. I have realised that PR was available just after I had pushed this out, then I put down a review on that :)

yamac-kurtulus committed:
Added myself and karsibali to Contributors list
Made suggested changes in PR #3353
Fixed "TR" in Translations.md so that it would be consistent with previous versions

yamac-kurtulus committed:
Merge branch 'master' into turkish-translation

yamac-kurtulus committed:
Merge branch 'master' into turkish-translation

yamac-kurtulus commented:
So I made the changes that @karsibali suggested in #3353. Added his name as well, as per request Should be good now.

Roshanjossey commented:
@yamac-kurtulus, this is brilliant. Great job on the discussions and incorporating changes from #3353.

Merging this 🎉
